packages/dds/tree/src/simple-tree/schemaFactory.ts

```diff
@@ -611,7 +611,7 @@ export class SchemaFactory<
 	 * Make a field of type identifier instead of the default which is required.
 	 */
 	public get identifier(): FieldSchema<FieldKind.Identifier> {
-		return createFieldSchema(FieldKind.Identifier, this.string);
+		return createFieldSchema(FieldKind.Identifier, [this.string, this.number]);
```

Comment:
Why this change?

Reply:
Reverted change, and updated TreeApi shortID return types